config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
+  # mount Rswag::Ui::Engine => '/api-docs'
+  # mount Rswag::Api::Engine => '/api-docs'
   resources :blogs
   resources :comments
   resources :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "/auth",to: "users#current_user"
+  post "/login",to: "sessions#create"
+  delete "/logout",to: "sessions#destroy"
+
```

Comment:
- [ ] When I list details about a post, I also want the comments for that post to be included as well. This starts with the routing being right and ends in the controller where you will include the :comments.For example for a blog and the comments, the routes could be edited to something like:

```
resources :blogs do
  resources :comments
end
```